code/style.css

```diff
@@ -0,0 +1,218 @@
+@import url("https://fonts.googleapis.com/css?family=Londrina+Outline");
+@import url("https://fonts.googleapis.com/css2?family=Vollkorn:wght@600&display=swap");
+/* Code for gradient background was borrowed from one of our classmates, Jonna! */
+
+html {
+  margin: 0;
+  scroll-behavior: smooth;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url(./img/bluepizza.jpg);
```

Comment:
Some browsers will not format that path because of lack of quotes. It's far more safe to define background property like this :

`background-image: url("./img/bluepizza.jpg");`